c2corg_ui/static/partials/map/map.html

```diff
@@ -13,7 +13,12 @@
     </div>
   </div>
 
-  <button type="button" ng-show="mapCtrl.edit" ng-click="mapCtrl.resetFeature()"
-          class="btn btn-primary btn-xs map-reset-feature" translate>Reset feature</button>
+  <button type="button" ng-show="mapCtrl.canReset()" ng-click="mapCtrl.resetFeature()"
+          class="btn btn-primary btn-xs map-reset-feature"
+          title="{{ 'Reset geometry' | translate }}" translate>Reset geometry</button>
```

Comment:
"Geometry" is also a GIS-related term. I don't think normal users know more about "geometries" than about "features".
If you think "feature" is not common enough (for me it is not a problem because the translation can be translated in some other way by translators), it would make more sense to use "track" instead.

I have not tested with point documents. Does your tool apply to such features?